src/components/Confirm/index.js

```diff
@@ -225,8 +228,30 @@ class Confirm extends Component  {
         const data = { challengeData, blob, language }
         uploadLiveVideo(data, token, this.onApiSuccess, this.onApiError)
       } else {
-        const data = { file: blob }
-        uploadLivePhoto(data, token, this.onApiSuccess, this.onApiError)
+        Promise.all([
+          promisifiedUploadLivePhoto(
+            {
+              file: capture.blob
+            },
+            token
+          ),
+          ...(capture.snapshot
```

Comment:
Can we wrap this in a named function to make it more readable?

Reply:
Yep! :D